.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: python
+
+python:
+   - "2.7"
+   - "3.6"
+
+matrix:
+ allow_failures:
+   - python: "3.6"
+
+install:
+   - python setup.py install
+   - pip install -r dev-requirements.txt
+
+script: pytest -v test
```

Comment:
For a pytest example, might as well include:
```yaml
script:
  - py.test -vv --cov=[REPOSITORY_NAME] --cov-report xml tests

after_script:
  - python-codacy-coverage -r coverage.xml
```
for code coverage reporting.

Reply:
added in 89182602d669a190bcacb05fe4502a2d974d2faf